lua/kubectl/views/filter/init.lua

```diff
@@ -31,7 +34,77 @@ local function save_history(input)
   state.filter_history = result
 end
 
+function M.filter_label()
+  local state = require("kubectl.state")
+  local instance = vim.deepcopy(state.instance)
+  local view, definition = views.view_and_definition(instance.resource)
+  local name, ns = view.getCurrentSelection()
+  if not name then
+    return
+  end
+
+  local resource = tables.find_resource(require("kubectl.state").instance.data, name, ns)
```

Comment:
Don't we already have the instance from above?

Reply:
fixed